Makefile

```diff
@@ -11,10 +11,10 @@ test_benchmark:
 	sbt "+ mleap-benchmark/test"
 
 test_xgboost_runtime:
-	sbt "mleap-xgboost-runtime/test"
+	sbt "+ mleap-xgboost-runtime/test"
 
 test_xgboost_spark:
-	sbt "mleap-xgboost-spark/test"
+	sbt "+ mleap-xgboost-spark/test"
```

Comment:
We can remove these separate tasks if we now add the `mleap-xgboost-runtime` and `mleap-xgboost-spark` subprojects in the `aggregatedProjects` list in MleapProject.scala`.

Reply:
That makes sense. What was the original reason for them to be separated ? I assume because xgboost wouldn't build on 2.12 ?